src/Microsoft.AspNetCore.Mvc.RazorPages/Internal/PageLoggerExtensions.cs

```diff
@@ -41,6 +42,10 @@ static PageLoggerExtensions()
                3,
                "Request was short circuited at page filter '{PageFilter}'.");
 
+            _redirectToPageResultExecuting = LoggerMessage.Define<string>(
+                LogLevel.Information,
+                1,
```

Comment:
@BrennanConroy says these should be unique within the package. If you make the ones you're adding/already editing unique I'll file a bug to fix the others.